spec/models/problem_spec.rb

```diff
@@ -475,4 +475,28 @@
       end
     end
   end
+
+  context "#url" do
+    let!(:original_protocol) { Errbit::Config.protocol }
+    let!(:original_host) { Errbit::Config.host }
+    let!(:original_port) { Errbit::Config.port }
+
+    subject { Fabricate(:problem) }
+
+    before(:each) do
+      Errbit::Config.protocol = "https"
+      Errbit::Config.host     = "memyselfandi.com"
+      Errbit::Config.port     = 8123
+    end
+
+    after(:each) do
+      Errbit::Config.protocol = original_protocol
+      Errbit::Config.host     = original_host
+      Errbit::Config.port     = original_port
+    end
+
+    it "uses the configured protocol" do
```

Comment:
Does it still work if you remove all the before/after hooks and instead use this kind of thing?
```ruby
allow(Errbit::Config).to receive(:protocol).and_return('https')
```